### PR TITLE
pytorch 1.2 compatibility

### LINKED ISSUE #1891: controllable_seq2seq.py fails lint.
https://circleci.com/gh/facebookresearch/ParlAI/7716?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

```
#!/bin/bash -eo pipefail
bash ./tests/lint_changed.sh
projects/controllable_dialogue/controllable_seq2seq/controllable_seq2seq.py:810:21: B006 Do not use mutable data structures for argument defaults. All calls reuse one instance of that data structure, persisting changes between them.
        wd_features=[],
                    ^
projects/controllable_dialogue/controllable_seq2seq/controllable_seq2seq.py:811:16: B006 Do not use mutable data structures for argument defaults. All calls reuse one instance of that data structure, persisting changes between them.
        wd_wts=[],
               ^
Exited with code 1
```

### PR DESCRIPTION
**Pull request by stephenroller**
**Patch description**
Same fix as #1905, but for controllable dialogue. Also fixes #1891, and updates wizard to handle the new `torch.bool` tensors.

**Testing steps**
CircleCI nightly GPU.

### CODE REVIEW

File: projects/controllable_dialogue/controllable_seq2seq/controllable_seq2seq.py [hunk 1]
```diff
@@ -807,35 +809,43 @@ def beam_search(
...
+
+        :return:
+            - beam_preds_scores : list of tuples (prediction, score) for each
+              sample in Batch
+            - n_best_preds_scores : list of n_best list of tuples (prediction,
+              score) for each sample from Batch
+            - beams : list of Beam instances defined in Beam class, can be used
+              for any following postprocessing, e.g. dot logging.
         """
+        if wd_features is None:
```
Comment (emilydinan) [inline projects/controllable_dialogue/controllable_seq2seq/controllable_seq2seq.py]: what is the point of changing the defaults to None and then immediately converting them to lists?

File: projects/controllable_dialogue/controllable_seq2seq/modules.py [hunk 2]
```diff
@@ -91,12 +88,16 @@ def __init__(
...
-        control_settings={},
+        control_settings=None,
     ):
-        """Initialize seq2seq model.
+        """
+        Initialize seq2seq model.
 
         See cmdline args in Seq2seqAgent for description of arguments.
         """
+        if control_settings is None:
```
Comment (emilydinan) [inline projects/controllable_dialogue/controllable_seq2seq/modules.py]: same here

File: projects/controllable_dialogue/controllable_seq2seq/controllable_seq2seq.py (hunk 1 above)
Comment (stephenroller) [reply projects/controllable_dialogue/controllable_seq2seq/controllable_seq2seq.py]: Lists are mutable objects, and default parameters are fixed. That means that if you do any modifications to that list, it changes the default argument for *all future calls*.

Comment (emilydinan) [reply projects/controllable_dialogue/controllable_seq2seq/controllable_seq2seq.py]: cuhraaaaaaaazy 😱

**Pull request merged**